src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs

```diff
@@ -13,33 +13,40 @@
     (fn [i] {:name (rand-nth ["rand_name_1" "rand_name_2" "rand_name_3" "rand_name_4"])
          :namespace (str "ns_s_" (inc i))
          :root-ent-type (str "r_e_t_" (inc i))
-         :workspaceName {:namespace (str "*ws_ns_" (inc i))
-                        :name (str "*ws_n_" (inc i))}
-         :methodStoreMethod {:namespace (str "*ms_ns_" (inc i))
-                            :name (str "*ms_n_" (inc i))
-                            :version (str "*ms_v_" (inc i))}
-         :methodStoreConfig {:namespace (str "*msc_ns_" (inc i))
-                             :name (str "*msc_n_" (inc i))
-                             :version (str "*msc_v_" (inc i))
+         :workspaceName {:namespace (str "ws_ns_" (inc i))
+                        :name (str "ws_n_" (inc i))}
+         :methodStoreMethod {:methodNamespace (str "ms_ns_" (inc i))
+                            :methodName (str "ms_n_" (inc i))
+                            :methodVersion (str "ms_v_" (inc i))}
+         :methodStoreConfig {:namespace (str "msc_ns_" (inc i))
+                             :name (str "msc_n_" (inc i))
+                             :version (str "msc_v_" (inc i))
                              }
-         :inputs {:i1 (str "*i_1_" (inc i))
-                  :i2 (str "*i_2_" (inc i))}
-         :outputs {:o1 (str "*o_1_" (inc i))
-                  :o2 (str "*o_2_" (inc i))}
-         :prerequisites {:p1 (str "*p_1_" (inc i))
-                  :p2 (str "*p_2_" (inc i))}})
+         :inputs {:i1 (str "i_1_" (inc i))
+                  :i2 (str "i_2_" (inc i))}
+         :outputs {:o1 (str "o_1_" (inc i))
+                  :o2 (str "o_2_" (inc i))}
+         :prerequisites {:p1 (str "p_1_" (inc i))
+                  :p2 (str "p_2_" (inc i))}})
     (range (rand-int 50))))
 
 
 
 
+(defn
+  stringify_map
+  [the_map]
+  (for [s (keys the_map)] (let [k s v (get the_map k)] (str k "," v " ; ")))
+  )
```

Comment:
Make this a 2-liner.

(defn stringify_map [the_map]
  (for ...

You also really don't need that let, just rename s to k and do the get inline.